bigbluebutton-html5/private/config/settings.yml

```diff
@@ -59,6 +59,7 @@ public:
     # the default logoutUrl matches window.location.origin i.e. bigbluebutton.org for demo.bigbluebutton.org
     # in some cases we want only custom logoutUrl to be used when provided on meeting create. Default value: true
     askForConfirmationOnLeave: true
+    enableWakeLock: false
```

Comment:
Should we leave only one of the configs?

Reply:
For whether the feature is enabled or not, let's mimic how other [recent] features do it -- see darkMode for example,

```
featureName
  enabled: Boolean
```